doc/source/configuration/wazuh.rst

```diff
@@ -0,0 +1,330 @@
+
+
+Manually provisioned VM
+-----------------------
+
+In case where you can’t use infra-vms to deploy your wazuh-manager VM but you want to configure
+host using kayobe, there are some tips (note that depending on your setup this don’t have to always apply):
+
+* Depending on preferences host have to be part of some group in inventory. ``infra-vms`` group still seems as best choice
+  You can use ``kayobe infra vm host configure`` to configure host in this case.
+  Bellow tips are based on assumption that infra-vm will be used.
+* user ``stack`` with password less sudo and accessible with ssh keys needs to be present on host.
+  It can be achieved in many different ways, depending on your setup.
+* lvm configuration should be placed in ``host_vars/<host_name>``
+* wazuh-manager host have to be part of ``infra-vms`` group (directly or as child)
+* network used on host needs to be defined in ``networks.yml`` and
+  if you have pre-alocated IP, it should be added to ``network-allocation.yml``.
+  For example, if using host with IP 10.10.224.5 in network 10.10.224.0/24 one have to add:
+
+
+``networks.yml``:
+
+.. code-block:: console
+
+    undercloud_admin_net_cidr: 10.10.224.0/24
+    undercloud_admin_net_allocation_pool_start: 10.10.224.3
+    undercloud_admin_net_allocation_pool_end: 10.10.224.200
+    undercloud_admin_net_gateway: 10.10.224.254
+
+
+``network-allocation.yml``:
+
+.. code-block:: console
+
+    undercloud_admin_net_ips:
+      nesmetprd01: 10.10.224.5
+
+Note that in this example network name is ``undercloud`` to demonstrate that this network isn't "standard" kayobe network.
+
+
+Deploying Wazuh Manager services
+================================
+
+Setup
+-----
+
+To install specific version modify wazuh-ansible entry in ``etc/kayobe/ansible/requirements.yml``:
+
+.. code-block:: console
+
+  roles:
+    - name: wazuh-ansible
+      src: https://github.com/stackhpc/wazuh-ansible
+      version: stackhpc
+
+Version above was tested and verified, but there is no reason to use not different one.
+
+Install the role:
+
+``kayobe control host bootstrap``
+
+
+Edit the playbook and variables to your needs:
+
+Wazuh manager configuration
+---------------------------
+
+Wazuh manager playbook is located in ``etc/kayobe/ansible/wazuh-manager.yml``.
+Running this playbook will:
+
+* generate certificates for wazuh-manager
+* setup and deploy filebeat on wazuh-manager vm
+* setup and deploy wazuh-indexer on wazuh-manager vm
+* setup and deploy wazuh-manager on wazuh-manager vm
+* setup and deploy wazuh-dashboard on wazuh-manager vm
+* copy certificates over to wazuh-manager vm
+
+Wazuh manager variables file is located in ``etc/kayobe/inventory/group_vars/wazuh-manager/wazuh-manager``.
+
+You may need to modify some of the variables, including:
+
+* wazuh_manager_ip
+
+
+.. note::
+
+    NOTE:
+    If you are using multiple environments you will need to move variable files to pertinent directory,
```

Comment:
I disagree with this. Moving the file will cause a diff with the base config that will make it harder to maintain, and lead to merge conflicts. It would be better to override variables where necessary, but ideally the defaults should work without changes in most cases.